P5/Source/Guidelines/en/TC-CriticalApparatus.xml

```diff
@@ -700,9 +717,8 @@ Barcelona, 1518, Carles Amorós. </bibl>
       </list></p>
     <p>Both the location-referenced and the double end-point methods may be used with either
         <term>in-line</term> or <term>external</term> apparatus, the former dispersed within the
-      base text, the latter held in some separate location, within or outside the document with the
-      base text. The parallel segmentation method does not use the concept of a base text and may
-      only be used for in-line apparatus.</p>
+      base text, the latter held in some separate location, within or outside the document containing the
+      base text. The parallel segmentation method may only be used for in-line apparatus.</p>
```

Comment:
Yes, good idea changing this. The parallel segmentation can absolutely use the concept of a base text.